test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_service_test.clj

```diff
@@ -744,6 +748,26 @@
              (is (= 200 (:status @response)))
              (is (= hello-body (:body @response))))))))))
 
+(deftest double-stop-test
+  (testing "if the stop lifecycle is called more than once, we handle that gracefully and quietly"
```

Comment:
Rather than changing the logging for all of the tests, would there be any way to have a logback file just used for this one test (could we start up trapperkeeper in this test with a `{:global logback.xml}` option?)